### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by mehdichaouch**
⚠️ before merge, Discord badge on README.md needs you have Enabled your servers widget or else the banners will not work properly (this can be found in Server Settings > Widget).

### CODE REVIEW

**Comment by mehdichaouch** [comment]
Discord badge on `README.md` needs you have Enabled your servers widget or else the banners will not work properly (this can be found in Server Settings > Widget).

File: nabmastodond/locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -169,11 +169,18 @@ msgstr ""
 "Une erreur est survenue lors de la récupération des données du compte : "
 
 #: templates/nabmastodond/settings.html:257
+#, fuzzy
+#| msgid ""
+#| "As we don't want Nabaztag to live in the cloud we're using <a "
+#| "href='https://en.wikipedia.org/wiki/Mastodon_(software)'>Mastodon</a> to "
+#| "connect rabbits (Mastodon is a sort of decentralized Twitter - and Open "
+#| "Source). Currently your Nabaztag is not connected to Mastodon."
```
Comment (f-laurens) [inline nabmastodond/locale/fr_FR/LC_MESSAGES/django.po:177]: Commented out stuff to be removed.

File: nabmastodond/locale/fr_FR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -182,10 +189,16 @@ msgstr ""
 "n'est pas connecté à Mastodon</b>."
 
 #: templates/nabmastodond/settings.html:259
+#, fuzzy
+#| msgid ""
+#| "First, <strong>create an account for your Nabaztag</strong> on a given "
+#| "Mastodon instance, you can find a directory of Mastodon instances <a href="
+#| "\"https://joinmastodon.org/\" target=\"_blank\">here</a>."
```
Comment (f-laurens) [inline nabmastodond/locale/fr_FR/LC_MESSAGES/django.po:196]: Commented out stuff to be removed.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -132,17 +131,17 @@ msgstr "animations visuelles Qualité de l'air"
 
 #: templates/nabweb/help.html:12
 msgid "Talking to the rabbit (speech recognition)"
-msgstr "Commnent parler au lapin (reconnaissance vocale)"
+msgstr "Comment parler au lapin (reconnaissance vocale)"
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:134]: Agreed - ugly typo indeed ;-)

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -162,7 +161,7 @@ msgid ""
 "target='_parent' href='/system-info/'>System information</a> tab of the "
 "configuration page."
 msgstr ""
-"You pouvez le faire proprement en utilisant les boutons <kbd class=\"btn-"
+"Vous pouvez le faire proprement en utilisant les boutons <kbd class=\"btn-"
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:164]: Agreed - ugly typo indeed ;-)

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -132,17 +131,17 @@ msgstr "animations visuelles Qualité de l'air"
...
 msgid ""
 "You need to push and hold the button on the rabbit's head long enough to "
-"hear a 'beep', then wait a couple of seconds and speak. When you are done "
+"hear a sound, then wait a couple of seconds and speak. When you are done "
 "just release the button."
 msgstr ""
 "Il vous faut maintenir le bouton sur la tête du lapin appuyé assez longtemps "
-"pour entendre le 'bip', attendre quelques secondes puis parler. Relachez "
-"quand vous avez terminé."
+"pour entendre un son, attendre quelques secondes puis parler. Relachez quand "
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:143]: I do not really agree with this change.
Even if 'beep'/'bip' is more colloquial than sound/son, I think it better coneys the idea that user has to wait for a specific (not any) sound.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -215,69 +214,98 @@ msgstr ""
...
 "L'opération à cœur ouvert pour transplanter la carte dans votre lapin. A ce "
 "stade, vous n'en avez probablement plus besoin ..."
 
-#: templates/nabweb/help.html:22
+#: templates/nabweb/help.html:23
+#, fuzzy
+#| msgid ""
+#| "<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-"
+#| "nabaztag/' rel='noopener noreferrer'>The eternal return of the Nabaztag</"
+#| "a>"
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:286]: Commented out stuff to be removed.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -914,3 +937,13 @@ msgstr "{} heures et 1 minute"
...
+
+#~ msgid ""
+#~ "The <a target='_blank' href='https://www.tagtagtag.fr/index_eng.html' "
+#~ "rel='noopener noreferrer'>Eternal Sunshine of the Nabaztag</a>"
+#~ msgstr ""
+#~ "L'<a target='_blank' href='https://www.tagtagtag.fr/index.html' "
+#~ "rel='noopener noreferrer'>éternel retour du Nabaztag</a>"
+
+#~ msgid "%(variant)s OS"
+#~ msgstr "Système d'exploitation %(variant)s"
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:949]: Commented out stuff to be removed.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -830,15 +853,15 @@ msgid ""
 "You can switch to the Pynab <b>master</b> branch to benefit from changes "
 "between releases, however this is not for the faint-hearted and requires "
 "good command of <a href='https://www.raspberrypi.org/documentation/remote-"
-"access/ssh/' target='_blanck' rel='noopener noreferrer'>SSH</a> and <a "
-"href='https://git-scm.com/docs/' target='_blanck' rel='noopener "
+"access/ssh/' target='_blank' rel='noopener noreferrer'>SSH</a> and <a "
+"href='https://git-scm.com/docs/' target='_blank' rel='noopener "
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:857]: Agreed - nasty typo indeed ;-(

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -830,15 +853,15 @@ msgid ""
...
 "noreferrer'>Git</a>."
 msgstr ""
 "Vous pouvez basculer sur la branche <b>master</b> de Pynab pour avoir les "
 "dernières nouveautés, mais ceci suppose que vous n'avez pas peur de <a "
 "href='https://www.raspberrypi.org/documentation/remote-access/ssh/' "
-"target='_blanck' rel='noopener noreferrer'>SSH</a> et de <a href='https://"
-"git-scm.com/docs/' target='_blanck' rel='noopener noreferrer'>Git</a>."
+"target='_blank' rel='noopener noreferrer'>SSH</a> et de <a href='https://git-"
+"scm.com/docs/' target='_blank' rel='noopener noreferrer'>Git</a>."
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:864]: Agreed - nasty typo indeed ;-(

File: nabweb/templates/nabweb/upgrade/index.html [hunk 10]
```diff
@@ -42,7 +42,7 @@ <h5 class="card-title">{% trans "Upgrade" %}</h5>
...
           {% if pynab.upstream_branch == "origin/release" %}
-          <p class="text-secondary">{% trans "You can switch to the Pynab <b>master</b> branch to benefit from changes between releases, however this is not for the faint-hearted and requires good command of <a href='https://www.raspberrypi.org/documentation/remote-access/ssh/' target='_blanck' rel='noopener noreferrer'>SSH</a> and <a href='https://git-scm.com/docs/' target='_blanck' rel='noopener noreferrer'>Git</a>." %}</p>
+          <p class="text-secondary">{% trans "You can switch to the Pynab <b>master</b> branch to benefit from changes between releases, however this is not for the faint-hearted and requires good command of <a href='https://www.raspberrypi.org/documentation/remote-access/ssh/' target='_blank' rel='noopener noreferrer'>SSH</a> and <a href='https://git-scm.com/docs/' target='_blank' rel='noopener noreferrer'>Git</a>." %}</p>
```
Comment (f-laurens) [inline nabweb/templates/nabweb/upgrade/index.html:45]: Agreed - nasty typo indeed ;-(

File: nabweb/templates/nabweb/help.html [hunk 11]
```diff
@@ -9,17 +9,19 @@ <h1 class="display-5">{% trans "Nabaztag help" %}</h1>
       <p class="lead">{% trans "On your rabbit" %}</p>
       <p>{% trans "<b>The " %}<a href='/help/weather' title='{% trans "The weather forecast visual animations" %}'>{% trans "weather forecast visual animations" %}</b></a>.</p>
       <p>{% trans "<b>The " %}<a href='/help/airquality' title='{% trans "The air quality visual animations" %}'>{% trans "air quality visual animations" %}</b></a>.</p>
-      <p id="asr"><b>{% trans "Talking to the rabbit (speech recognition)" %}</b><br>{% trans "You need to push and hold the button on the rabbit's head long enough to hear a 'beep', then wait a couple of seconds and speak. When you are done just release the button." %}<br>{% trans "The speech processing is done locally and the answer should come shortly!" %}</p>
+      <p id="asr"><b>{% trans "Talking to the rabbit (speech recognition)" %}</b><br>{% trans "You need to push and hold the button on the rabbit's head long enough to hear a sound, then wait a couple of seconds and speak. When you are done just release the button." %}<br>{% trans "The speech processing is done locally and the answer should come shortly!" %}</p>
```
Comment (f-laurens) [inline nabweb/templates/nabweb/help.html:12]: I do not really agree with this change.
Even if 'beep' is more colloquial than sound, I think it better conveys the idea that user has to wait for a specific (not any) sound.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -215,69 +214,98 @@ msgstr ""
 
 #: templates/nabweb/help.html:17
 msgid "Subscribe to the newsletter."
-msgstr "Abonnez vous à la lettre d'information."
+msgstr "Abonnez-vous à la lettre d'information."
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:217]: Agreed - ugly typo indeed ;-)

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -215,69 +214,98 @@ msgstr ""
...
 msgid ""
+"<a target='_blank' href='https://dsc.gg/nabaztagtagtag' rel='noopener "
+"noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>"
+msgstr ""
+"<a target='_blank' href='https://dsc.gg/nabaztag' rel='noopener "
+"noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>"
+
+#: templates/nabweb/help.html:18
+msgid "Modern place to discuss."
+msgstr "Salon de discussoin."
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:229]: I do not think we should add a reference to Discord in the help page.
If needed (will it live long?..), it should rather be referenced from the tagtagtag.fr project page.

File: nabweb/templates/nabweb/help.html [hunk 14]
```diff
@@ -9,17 +9,19 @@ <h1 class="display-5">{% trans "Nabaztag help" %}</h1>
...
       <p><b>{% trans "Base LED pulse color" %}</b> {% trans "indicates the rabbit's network connection state" %}<br>{% blocktrans %}<span class="badge badge-pill badge-info">Fuchsia</span> means <i>all is fine</i>: rabbit has an IP address on your local network, and access to the Internet.<br><span class="badge badge-pill badge-warning">Orange</span> means <i>not so good</i>: rabbit has a local IP address, but no Internet connection (check your router).<br><span class="badge badge-pill badge-danger">Red</span> means <i>real bad</i>: rabbit has no IP address (check your WiFi access point / DHCP server).{% endblocktrans %}</p>
       <hr class="my-4">
       <p class="lead">{% trans "On the Internet" %}</p>
       <p><b>{% trans "<a target='_blank' href='https://tinyletter.com/nabaztag' rel='noopener noreferrer'>Nabaztag is back!</a>" %}</b><br>{% trans "Subscribe to the newsletter." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://dsc.gg/nabaztagtagtag' rel='noopener noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>" %}</b><br>{% trans "Modern place to discuss." %}</p>
```
Comment (f-laurens) [inline nabweb/templates/nabweb/help.html:18]: I do not think we should add a reference to Discord in the help page.
If needed (will it live long?..), it should rather be referenced from the tagtagtag.fr project page.

File: nabweb/locale/fr_FR/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -215,69 +214,98 @@ msgstr ""
...
+msgid ""
+"<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' "
+"rel='noopener noreferrer'>The eternal return of the Nabaztag</a>"
+msgstr ""
+"<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' "
+"rel='noopener noreferrer'>L'éternel retour du Nabaztag</a>"
+
+#: templates/nabweb/help.html:24
+msgid "Collection 2021-2022"
+msgstr "Collection 2021-2022"
```
Comment (f-laurens) [inline nabweb/locale/fr_FR/LC_MESSAGES/django.po:308]: I do **not** agree with this change, which basically reverts the changes that were made in PR https://github.com/nabaztag2018/pynab/pull/298 [remove direct references to Ulule campaigns (which come and go...) and replace them by a reference to the tagtagtag.fr project page (which itself references the Ulule campaigns)].

File: nabweb/templates/nabweb/help.html [hunk 16]
```diff
@@ -9,17 +9,19 @@ <h1 class="display-5">{% trans "Nabaztag help" %}</h1>
...
       <p><b>{% trans "<a target='_blank' href='https://twitter.com/nabaztagtagtag' rel='noopener noreferrer'>Nabaztag on Twitter</a>" %}</b></p>
       <p><b>{% trans "The <a target='_blank' href='https://www.instructables.com/member/tagtagtag/' rel='noopener noreferrer'>TagTagTag assembly instructions</a>" %}</b><br>{% trans "The open heart operation to transplant the card into your rabbit. At this stage you probably do not need this anymore..." %}</p>
-      <p><b>{% trans "The <a target='_blank' href='https://www.tagtagtag.fr/index_eng.html' rel='noopener noreferrer'>Eternal Sunshine of the Nabaztag</a>" %}</b><br>{% trans "The TagtagTag project..." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://fr.ulule.com/le-retour-du-retour-du-nabaztag/' rel='noopener noreferrer'>Nabaztag is BackBack</a>" %}</b><br>{% trans "Some project history..." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' rel='noopener noreferrer'>The eternal return of the Nabaztag</a>" %}</b><br>{% trans "Collection 2021-2022" %}</p>
```
Comment (f-laurens) [inline nabweb/templates/nabweb/help.html:24]: I do not agree with this change, which basically reverts the changes that were made in PR #298 [remove direct references to Ulule campaigns (which come and go...) and replace them by a reference to the tagtagtag.fr project page (which itself references the Ulule campaigns)].

**Comment by f-laurens** [comment]
As the original implementer (and maintainer so far...) of the Pynab embedded help page, I have made a number of comments on the proposed changes to this page:
- I do agree with the typo fixes (Thanks a lot!)
- but I disagree with some other changes:
  -   link changes that reverse the changes made in  PR #298 [remove direct references to Ulule campaigns (which come and go...) and replace them by a reference to the tagtagtag.fr project page (which itself references the Ulule campaigns)];
  -   addition of a link to Discord: it should rather be referenced from the tagtagtag.fr project page.
       Also, IMHO, Discord is ugly (and its current contents are a mess...), and should not be promoted to replace the Tag:tag:tag user forum.

**Comment by nabaztag2018** [comment]
Hello,
I agree with @f-laurens regarding the links, we should refer to tagtagtag.fr web page rather than directly to the campaign or the Discord or else. It's easier to modify these references if needed rather than pushing a new release.
O.

**Comment by mehdichaouch** [comment]
> Hello, I agree with @f-laurens regarding the links, we should refer to tagtagtag.fr web page rather than directly to the campaign or the Discord or else. It's easier to modify these references if needed rather than pushing a new release. O.

OK, I will do changes tomorrow.

**Comment by mehdichaouch** [comment]
@nabaztag2018 could you explain me why did you merge #310 which is a copycat of the changed I have done in this MR (on my free time)?
Did I missed something on how to contribute in pynab?

**Comment by nabaztag2018** [comment]
Hello @mehdichaouch ,
Sorry, I thought that #310 was a PR based on yours but without the direct links to content that could change in the future, like Discord, Ulule campaign and stuff like that. It's easier to change the website rather than pushing new release.
Did I miss something ?

**Comment by mehdichaouch** [comment]
> Hello @mehdichaouch , Sorry, I thought that #310 was a PR based on yours but without the direct links to content that could change in the future, like Discord, Ulule campaign and stuff like that. It's easier to change the website rather than pushing new release. Did I miss something ?

IMO I don't understand why merging a copycat content, when [I said](https://github.com/nabaztag2018/pynab/pull/308#issuecomment-1021654349) that I will do the changes.
When I need a code made by someone I grab the commit hash to my branch, like here :
image
I don't copy the commit content and make it mine !

Idem to @f-laurens I don't understand why did you make a clone of my changes (typo fix), because it was not a critical issue, only typo. What was your motivation to make #310, because I don't understand this action.

Merci

**Comment by f-laurens** [comment]
@mehdichaouch: as specified in #310, the intent was to propose an alternative to #308 containing only the essential (critical?) changes (typo and link fixes), for which you were duly credited. That is all.

**Pull request closed without merging**